jdbcSource/src/test/java/io/vantiq/extsrc/jdbcSource/TestJDBC.java

```diff
@@ -149,7 +150,8 @@ public static void debugSetup() throws VantiqSQLException {
     }
     
     */
-    
+
+    @SuppressWarnings("PMD.JUnit4TestShouldUseAfterAnnotation")
```

Comment:
This is the warning that methods named `tearDown` should be marked with `@After`.  In JUnit3, methods with that name were always run after the class was run.  Annotations allowed for more flexibility, and this is a warning for transitions.  We don't fall into that category (and we're using this as an `@AfterClass`, so we'll skip this warning.  

Could also rename the method.  But this is a nice example...